tgui/packages/tgui/styles/main.scss

```diff
@@ -36,6 +36,7 @@
 @include meta.load-css('./components/ProgressBar.scss');
 @include meta.load-css('./components/RoundGauge.scss');
 @include meta.load-css('./components/Section.scss');
+@include meta.load-css('./components/SectionWithTitleOverflow.scss');
```

Comment:
Probably put this in `./components/goonstation/SectionWithTitleOverflow.scss`, for the sake of delimiting what we have as goonstation custom components and not.